packages/layout/src/components/cards/common/views/address/addressPage.tsx

```diff
@@ -34,7 +34,12 @@ const AddressPage: React.FC<AddressPageProps> = ({
 	const addressLookupProvider = new ExperianAddressLookupProvider(addressAPI);
 
 	return (
-		<Content type={cardType} typeName={cardTypeName} title={i18n.title}>
+		<Content
+			type={cardType}
+			typeName={cardTypeName}
+			title={i18n.title}
+			sectionTitle={`Edit ${cardTypeName ? cardTypeName : cardType}`}
```

Comment:
All of these strings should be coming from `i18n.something`. If one day someone decides that "Change" is better than "Edit", or we need to make it multi-lingual like the corporate website, then this is still hard-coded.

Reply:
Added a section title to the i18n file for all available cards. I've had to change the API of some components to allow this to be passed through.